pkg/cmd/secrets.go

```diff
@@ -210,7 +210,7 @@ func downloadSecrets(cmd *cobra.Command, args []string) {
 		format = "json"
 	}
 
-	validFormats := []string{"json", "env"}
+	validFormats := []string{"json", "env", "yaml"}
```

Comment:
NIT: use typealias for this?

Reply:
Can you add an example of what you mean? I'm not entirely sure

Reply:
Not sure if we already talked about this or not, but this is a pretty good read: https://alehatsman.com/posts/golang_type_declaration_abuse.html

Reply:
Added a type `SecretsFormat`